.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Please remove this file (just keep a local copy) since we cannot have configs for each editor in here :)

Reply:
Sounds good, removed!